applications.go

```diff
@@ -42,14 +42,20 @@ type PermissionsType struct {
 	Execute []string `json:"EXECUTE" mapstructure:"EXECUTE" yaml:"EXECUTE" hcl:"EXECUTE"`
 }
 
+type NotificationsType map[string]Notification
+
+type Notification interface {
```

Comment:
You are correct. I deleted Notification interface, changed NotificationsType to ma[string]interface.
There is already a func (notifications *NotificationsType) ValidateAppNotification() that is going to validate the structure of the notifications, since it's composed of slices for notifications and string for application key. I added it to UpdateApplicationNotifications so we are ensuring it validates before making changes.